tests/phpunit/api/v3/ContactTest.php

```diff
@@ -1573,6 +1576,38 @@ public function testCreateIndividualWithNameEmail() {
     $this->callAPISuccess('contact', 'delete', $contact);
   }
 
+  /**
+   * Verify that attempt to create individual contact just email succeeds.
+   *
+   *
+   * @dataProvider versionThreeAndFour
+   * @param int $version
+   *
+   * @throws API_Exception
+   * @throws CRM_Core_Exception
+   */
+  public function testCreateIndividualWithJustEmail($version) {
+    $this->_apiversion = $version;
+    $params = [
+      'contact_type' => 'Individual',
+      'email' => 'garlic@example.com',
+    ];
```

Comment:
```suggestion
    $params = ['contact_type' => 'Individual'];
    if ($version == 3) {
      $params['email'] = 'garlic@example.com';
    }
    else {
      $params['api.Email.create'] = ['location_type_id' => 1, 'email' => 'garlic@example.com'];
    }
```

Reply:
Really we don't need to add test cover for v3 - the test was to demonstrate the gap/ difference

Reply:
Ok, let's just see if this passes.